test/models/billing/limiter_test.rb

```diff
@@ -208,16 +208,16 @@ def current_products
       end
 
       it "returns the usage count for the specified exhausted limit" do
-        tracker = FactoryBot.create(:tracker, team: team, interval: "month", duration: 1)
+        tracker = FactoryBot.create(:tracker, team_id: team.id, interval: "month", duration: 1)
         FactoryBot.create(:count, tracker: tracker, action: "created", name: "Blah", count: 3)
 
         Billing::Usage::ProductCatalog.stub(:all_products, all_products) do
-          assert_equal limiter.exhausted_usage_for(limit, :create, "Blah"), 3
+          assert_equal 3, limiter.exhausted_usage_for(limit, :create, "Blah")
```

Comment:
<img width="1103" alt="Screenshot 2025-03-21 at 11 02 20 AM" src="https://github.com/user-attachments/assets/8c09a11a-e8d9-421b-bcbf-9d2a24e31b35" />

Updated so the test correctly returns what was `expected` vs. `actual`